### DISCUSSION TITLE
Why is `colour.xy_to_XYZ` definition producing Y=1?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1130] [Q&A]

**Comment by thomasmansencal** [comment]
Hi,

As you edited and closed the question, I suppose you found out the answer! We picked one as it is common practice to pick unity when Luminance is unknown.

Cheers,

Thomas

**Reply by blue-j** [reply]
How embarrassing!  I just didn't feel confident that my question made sense until I did more research, and I tried to delete it entirely and found out I couldn't.... I promise I try to work things out myself for quite some time before asking others for support.  You are the best, Thomas, thank you!!  I have indeed seen the convention of making Y=1.  I was motivated because I thought other calculations needed XYZ to be as Apple represents them, but now I think they don't.  : )     - J
